hw/top_earlgrey/dv/vendor_chip_sim_cfg_sample.hjson

```diff
@@ -29,5 +29,10 @@
               "--cores-root my-custom-path-1",
               "--cores-root my-custom-path-2"]
     }
+    // Example 3: Change where same design files are sourced from - open source vs partner.
+    {
+      name: design_source
+      value: partner
```

Comment:
can you change this to `vendor`?

Reply:
Done.

Reply:
we have used `fileset_partner` in top-level. should we replace all of them?

https://github.com/lowRISC/opentitan/blob/840b59a02e86a6421c6ceeddfa0bcddf1fdc2efa/hw/top_earlgrey/top_earlgrey.core#L49

Reply:
Thanks @weicaiyang for pointing out. I have now updated those in this PR.